src/encoded/static/components/search.js

```diff
@@ -1386,9 +1389,10 @@ const ResultBrowser = (props) => {
 };
 
 ResultBrowser.propTypes = {
-    files: PropTypes.array.isRequired, // Array of files whose browser we're rendering
-    assembly: PropTypes.string.isRequired, // Filter `files` by this assembly
-    datasets: PropTypes.array.isRequired, // One or more '/dataset/ENCSRnnnXXX/' that files belong to
+    files: PropTypes.array, // Array of files whose browser we're rendering
+    assembly: PropTypes.string, // Filter `files` by this assembly
+    region: PropTypes.string, // Filter `files` by this assembly
+    datasets: PropTypes.array, // One or more '/dataset/ENCSRnnnXXX/' that files belong to
     limitFiles: PropTypes.bool, // True to limit browsing to 20 files
```

Comment:
These propTypes are all coming up with eslint errors, because AirBnB requires defaultProps if you don’t have `isRequired`.